lib/src/main/kotlin/WeatherSensorService/Internal/MockWeatherSensorReader.kt

```diff
@@ -0,0 +1,69 @@
+ //
+ // SpinDance
+ // Weather Sensor Service
+ //
+ // Copyright © 2022 SpinDance. All rights reserved.
+ //
+ 
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import java.util.Timer
+import kotlin.concurrent.*
+import kotlin.random.Random
+import java.time.LocalDateTime
+
+private class Constants {
+    companion object {
+        var READER_INTERVAL: UInt = 1U
```

Comment:
Since this is a constant, it should be immutable (`val`)

Reply:
Also, since the type is declared as UInt, is the `U` in `1U` necessary? Seems like it would not be.

Reply:
It won't build without the U. It says: The integer literal does not conform to the expected type UInt

Reply:
OK, thanks for that. In Swift the compiler would infer. Silly Kotlin...